guides/v2.2/config-guide/cli/config-cli-subcommands-index.md

```diff
@@ -120,23 +120,21 @@ Reindexing all indexers can take a long time for stores with large numbers of pr
 
 ### Reindex in parallel mode {#config-cli-subcommands-index-reindex-parallel}
 
-As of Magento 2.2.5, indexers are scoped and multi-threaded to support reindexing in parallel mode. This feature reduces processing time. It parallelizes by the indexer's dimension and executes across multiple threads.
+As of Magento 2.2.6, indexers are scoped and multi-threaded to support reindexing in parallel mode. This feature reduces processing time. It parallelizes by the indexer's dimension and executes across multiple threads.
 
 The following indexes can be run in parallel mode:
-- Catalog Search Fulltext can be paralleled by store views.
-- Category Product can be paralleled by store views.
+- Catalog Search Fulltext is paralleled by store views.
+- Category Product is paralleled by store views.
+- Catalog Price can be paralleled by website and customer groups.
 
-To reindex in parallel mode, run the reindex command using the environment variable `MAGE_INDEXER_THREADS_COUNT`. This variable sets the number of threads for the reindex processing.
+To reindex in parallel mode, run the reindex command using the environment variable MAGE_INDEXER_THREADS_COUNT or add an environment variable to env.php. This variable sets the number of threads for the reindex processing.
 
 For example, the following command runs the Catalog Search Fulltext indexer across three threads:
 
 ```bash
 MAGE_INDEXER_THREADS_COUNT=3 php -f bin/magento indexer:reindex catalogsearch_fulltext
 ```
 
-{:.bs-callout .bs-callout-info}
```

Comment:
Why was this note removed? @vzabaznov is this a correct removal?

Reply:
because it's not relevant information since 2.2.6 version, MAGETWO-93319

Reply:
I can confirm that its correct, since CLI command logic is upgraded we can bypass any limitation in saving variable in env.php. Andrii made correct changes

Reply:
Great! 👍